src/components/StyledSelect.js

```diff
@@ -213,26 +204,12 @@ export default class StyledSelect extends React.Component {
     }
   }
 
-  /** Return the value if component is controlled, otherwise undefined */
-  getValue() {
-    if (this.props.value === undefined) {
-      return undefined;
-    }
-
```

Comment:
This was a mistake .....this errors was made when I was going through the file ...I actually thought I undid all of these.....but when I commited my laptop was almost shutting down so had to just commit like that.....will retrace all of these by tommorow was in the process of doing that before my laptop went off